Jint/Collections/HashHelpers.cs

```diff
@@ -0,0 +1,15 @@
+namespace Jint.Collections
+{
+    internal static class HashHelpers
```

Comment:
private class in the StringDictionarySlim.cs file?

Reply:
Yes, will move, just followed corefxlabs layout for files for easier checking while they were still tweaking.